src/components/Attachment/Attachment.tsx

```diff
@@ -94,6 +136,34 @@ class Attachment extends UIComponent<Extendable<AttachmentProps>, any> {
       </ElementType>
     )
   }
+
+  protected actionHandlers: AccessibilityActionHandlers = {
+    performClick: event => this.handleKeyboardClick(event),
+  }
+
+  private handleKeyboardClick = e => {
+    e.stopPropagation()
+    this.handleClick(e)
+  }
```

Comment:
I would not move this to separate method, just leave it as

```js
performClick: event => {
   event.stopPropagation()
   this.handleClick()
}
```
I think ```handleKeyboardClick``` can be confusable + too many things like ```handleSmth``` already here